odoo/addons/base/models/ir_model.py

```diff
@@ -523,9 +523,9 @@ class IrModelFields(models.Model):
     copied = fields.Boolean(string='Copied',
                             compute='_compute_copied', store=True, readonly=False,
                             help="Whether the value is copied when duplicating a record.")
-    related = fields.Char(string='Related Field', help="The corresponding related field, if any. This must be a dot-separated list of field names.")
+    related = fields.Char(string='Related Field Name', help="The corresponding related field, if any. This must be a dot-separated list of field names.")
```

Comment:
I don't think that is less confusing. 'related' is not a field, it is actually a sequence of field names (e.g. `company_id.currency_id`)

Reply:
Lets go for `Related Field Name Sequence` then ?

Reply:
Seriously?  That's not better either.  If a user does not understand what "related" means in Odoo, this does not help.  If you expect the user to know what "related" means, then I suggest "Related Field Definition", because the value of that field is how the field is defined as a related field.  Otherwise, I don't know.

Reply:
"Related Field Definition" definitely sounds better than whatever I was trying to come up with 😂